tests/functional/pod_and_daemons/test_csi_logs_rotation.py

```diff
@@ -0,0 +1,93 @@
+import time
+import logging
+
+from ocs_ci.framework.testlib import BaseTest
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.constants import OPENSHIFT_STORAGE_NAMESPACE
+from ocs_ci.framework.pytest_customization.marks import brown_squad, tier1
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+
+log = logging.getLogger(__name__)
+LOGS_DIR_NAME = "/var/lib/rook/openshift-storage.cephfs.csi.ceph.com/log/node-plugin/"
+CURRENT_LOG_NAME = "csi-cephfsplugin.log"
+WAIT_FOR_ROTATION_TIME = 1200  # seconds
+SLEEP_BETWEEN_TRIES = 300  # seconds
+
+
+@brown_squad
+@tier1
```

Comment:
I wouldn't consider this test tier1 since it does not validate basic product functionality. Suggest changing to tier2

Reply:
@ebenahar Done.